qutebrowser/config/configdata.py

```diff
@@ -1241,9 +1242,13 @@ def data(readonly=False):
              "Default monospace fonts."),
 
             ('completion',
-             SettingValue(typ.Font(), DEFAULT_FONT_SIZE + ' ${_monospace}'),
+             SettingValue(typ.Font(),  DEFAULT_FONT_SIZE + ' ${_monospace}'),
              "Font used in the completion widget."),
 
+             ('completion.category',
+              SettingValue(typ.Font(),  'bold' + '${completion}'),
```

Comment:
You can simply use `'bold ${completion}'` here instead of concatenating the two strings